tests/lib/test_config.py

```diff
@@ -1,176 +1,214 @@
 # pylint: disable=missing-function-docstring disable=missing-module-docstring
 
-from unittest.mock import call
+import unittest
+from unittest.mock import call, patch
 
-import pytest
-
-from nestor_api.config.config import Configuration
+import nestor_api.lib.config as config
 from nestor_api.errors.config.aggregated_configuration_error import AggregatedConfigurationError
 from nestor_api.errors.config.app_configuration_not_found_error import AppConfigurationNotFoundError
-import nestor_api.lib.config as config
-import nestor_api.lib.io as io
-
-
-def test_change_environment(mocker):
-    spy = mocker.patch.object(io, "execute")
-
-    config.change_environment("environment", "path/to/config")
-
-    spy.assert_has_calls(
-        [
-            call("git stash", "path/to/config"),
-            call("git fetch origin", "path/to/config"),
-            call("git checkout environment", "path/to/config"),
-            call("git reset --hard origin/environment", "path/to/config"),
-        ]
-    )
-
-
-@pytest.mark.usefixtures("config")
-def test_create_temporary_config_copy(mocker):
-    spy = mocker.patch.object(io, "create_temporary_copy", return_value="/temporary/path")
-
-    path = config.create_temporary_config_copy()
 
-    spy.assert_called_once_with("/fixtures-nestor-config", "config")
-    assert path == "/temporary/path"
+FAKE_ENV = {
+    "NESTOR_CONFIG_PATH": "/fixtures-nestor-config",
+    "NESTOR_PRISTINE_PATH": "/fixtures-nestor-pristine",
+    "NESTOR_WORK_PATH": "/fixtures-nestor-work"
+}
+
+
+@patch('nestor_api.lib.config.io', autospec=True)
+class TestConfigLibrary(unittest.TestCase):
+    def test_change_environment(self, io_mock):
+        config.change_environment("environment", "path/to/config")
+
+        io_mock.execute.assert_has_calls(
+            [
+                call("git stash", "path/to/config"),
+                call("git fetch origin", "path/to/config"),
+                call("git checkout environment", "path/to/config"),
+                call("git reset --hard origin/environment", "path/to/config"),
+            ]
+        )
 
+    @patch.dict('os.environ', FAKE_ENV)
+    def test_create_temporary_config_copy(self, io_mock):
+        io_mock.create_temporary_copy.return_value = "/temporary/path"
 
-def test_get_app_config(mocker):
-    mocker.patch.object(Configuration, "get_config_path", return_value="tests/__fixtures__/config")
+        path = config.create_temporary_config_copy()
 
-    app_config = config.get_app_config("backoffice")
+        io_mock.create_temporary_copy.assert_called_once_with(
+            "/fixtures-nestor-config",
+            "config"
+        )
+        assert path == "/temporary/path"
```

Comment:
Since we are now using unittest, I think we could use the more classical syntax [`self.assertEqual()`](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertEqual)

```suggestion
        self.assertEqual(path, "/temporary/path")
```